internal/pkg/manifest/validate.go

```diff
@@ -39,6 +39,10 @@ const (
 
 	// Tracing vendors.
 	awsXRAY = "awsxray"
+
+	// deployment strategies
+	defaultStrategy  = "default"
+	recreateStrategy = "recreate"
```

Comment:
Can we move these consts to the `workload_ecs.go` file? and rename:
```suggestion
	// deployment strategies
	ecsDefaultRollingUpdateStrategy  = "default"
	ecsRecreateRollingUpdateStrategy = "recreate"
```